website/docs/components/form/super-select/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,192 @@
+## Usage
+
+### When to use
+
+- When the list of options is large and search is required to help filter the list.
+- When multiple selections are possible.
+- When list items are displayed in groups.
+- When you require complex structured content to be displayed within the list (and trigger).
+
+### When not to use
+
+- To display a list of links (or buttons) in a menu, consider the [HDS Dropdown](/components/dropdown) component instead.
+- When a simple select is needed for a form, consider the [HDS Select](/components/form/select) form element.
+- As a means to filter an array of objects within a data set; e.g., when presented in a tabular format or list. Instead, consider other [filtering patterns](/patterns/filter-patterns).
+
+## Components
+
+### Single
+
+Selection is limited to a single option item with structured content.
+
+![Single selection example with list open](/assets/components/form/super-select/super-select-single-variant-text.png =504x*)
+
+### Multiple
+
+Multiple options can be selected.
+
+![Multiple selection example with list open](/assets/components/form/super-select/super-select-multiple-variant.png =504x*)
+
+## Trigger value states
+
+**Empty**
+
+By default, the trigger is empty, indicating that nothing is selected.
+
+![Showing an empty trigger, indicating to the user there is no option selected yet](/assets/components/form/super-select/super-select-trigger-empty.png)
+
+**Placeholder text**
+
+Placeholder text can be added to the trigger to provide the user additional helpful context. However this information should not be required for them carry out a task.
+
+![Showing placeholder text inside of the trigger, still indicating no selection is made, but with some additional guidance](/assets/components/form/super-select/super-select-trigger-placeholder-text.png)
+
+**Filled**
+
+When one or more values are selected, the trigger is populated with the selected value(s). One or more values can be pre-selected before user interaction, as needed.
+
+![Showing preselected options in both a single and multiple variant. This helps show something has been preselected previously and the user can update their options](/assets/components/form/super-select/super-select-trigger-filled.png)
+
+## Before and after options
+
+The before and after options are areas above and below the list, independent from the list of options. Predefined components (like search) or custom content can occupy these spaces.
+
+### BeforeOptions
+
+BeforeOptions allows for either a search field or custom content to be displayed before the options list. Applicable to both Single and Multiple components.
+
+!!! Warning
+
+Currently, the Multiple component only supports search in the trigger and is considered an accessibility issue (nested interactive). This is something we are working to improve in collaboration with the PowerSelect team.
```

Comment:
Maybe consider swapping the order here, however as I'm writing this I don't think this is actually what you're referring to. Grain of salt here, but this first sentence was hard to read.
```suggestion
Currently, only the Multiple component supports search in the trigger which is considered an accessibility issue (nested interactive). This is something we are working to improve in collaboration with the PowerSelect team.
```

Reply:
I'm going to think through this one a bit more...

Reply:
Ok I overhauled this and think I've managed to make it much clearer

"Currently, the Multiple component includes search within an area of the trigger instead of displaying it in the beforeOptions. This causes a nested interaction which is an accessibility violation. We are in the process of working with the Ember Power Select maintainers to resolve this issue and make it the same as in the Single component."